app/uk/gov/hmrc/gform/dms/DmsHtmlSubmission.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.gform.dms
 
 import play.api.libs.json.{ Format, Json }
 
-case class DmsHtmlSubmission(html: String, metadata: DmsMetadata)
+case class DmsHtmlSubmission(b64html: String, attachments: List[Array[Byte]], metadata: DmsMetadata)
```

Comment:
It seems that the html should be sent in b64 format but that was not clear to me until I dug into the code. So making this explicit in the name here.